code/math/curve.cpp

```diff
@@ -9,6 +9,61 @@ int curve_get_by_name(const SCP_string& in_name) {
 	return find_item_with_field(Curves, &Curve::name, in_name);
 }
 
+static int curve_inline_def(){
+	//The curve needs a unique identifier. Build this with semicolon + filename + semicolon + character in file
+	SCP_string curve_name = ';' + SCP_string(Current_filename) + ';' + std::to_string(Mp - Parse_text);
+
+	int resulting_curve = static_cast<int>(Curves.size());
+	Curve& new_curve = Curves.emplace_back(curve_name);
+
+	do {
+		curve_keyframe& new_keyframe = new_curve.keyframes.emplace_back();
+		stuff_float(&new_keyframe.pos.x);
+		stuff_float(&new_keyframe.pos.y);
+		required_string(")");
+
+		bool found_interpolation = true;
+		int interpolation_mode = optional_string_one_of(4, "--", "-|", "-/", "/-");
```

Comment:
Alternate token suggestions, feel free to take them or leave them:
* Constant: `=`
* Square with lead in: `/[]`
* Square with lead out: `[]\`

Reply:
I'd prefer to keep the other ones for one main reason:
They're all the same length, which makes lining them up in an editor easier, as well as simplifies parsing.